reco_utils/azureml/azureml_designer_modules/module_specs/stratified_splitter.yaml

```diff
@@ -1,16 +1,20 @@
-name: Splitter Stratified
-id: efd1af54-0d31-42e1-b3d5-ce3b7c538705
-version: 1.1.0
-category: Recommenders
+moduleIdentifier: 
+  namespace: microsoft.com/cat
+  moduleName: Splitter Stratified
```

Comment:
Maybe it should be "Stratified Splitter".

Reply:
Good catch!